packages/api/cms-api/src/file-uploads/file-uploads.config.ts

```diff
@@ -8,5 +8,12 @@ export interface FileUploadsConfig {
     download?: {
         public?: boolean;
         secret: string;
+        /**
+         * Allows disabling the `downloadUrl` and `imageUrl` field resolvers.
+         * Use this when using file uploads without GraphQL.
+         *
+         * @default true
+         */
+        resolver?: boolean;
```

Comment:
is it clear which resolver?

should it be named more explicit? createFieldResolver or so?

Reply:
I chose resolver because the file was called resolver, but we can improve the naming:

- fieldResolvers
- createFieldResolvers
- disableFieldResolvers

Reply:
maybe createFieldResolvers would be better, but resolver is also ok

Reply:
Let's be explicit: 0a836367996730df220e9f167f59d1c10e290586